README.md

```
@@ -78,23 +78,23 @@ the **RUNTIME** requirements.*
-   | main branch, O3DE_2310 tag | Release 2310.x                                          | N/A                                                               | Release 2310.x                       |
+   | o3de-rgl-gem                   | [o3de/o3de-extras](https://github.com/o3de/o3de-extras) | [o3de](https://github.com/o3de/o3de) |
+   | ------------------------------ | ------------------------------------------------------- | ------------------------------------ |
+   | `o3de-2505` branch             | Release 2505.x                                          | Release 2505.x                       |
+   | `main` branch, `O3DE_2409` tag | Release 2409.x                                          | Release 2409.x                       |
+   | `main` branch, `O3DE_2310` tag | Release 2310.x                                          | Release 2310.x                       |
 
-   **_Note:_** _This table describes build combinations that are guaranteed to work. There may exist other working build
-   combinations._
+  The latest versions of the Gem (`development` branch) are expected to work with the latest versions of O3DE and ROS2 Gem (Release 2510.x and newer), but they have not been fully tested yet.
+
+   **_Note:_** _This table describes build combinations that are guaranteed to work. There may exist other working build combinations._
```

markdownlint-cli2 (0.21.0)

[warning] Emphasis style
Expected: asterisk; Actual: underscore

(MD049, emphasis-style)